docs/modules/install/proc-install-operator.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// assemblies/assembly-operator-install.adoc
+
+[id='install-cluster-operator-{context}']
+= Installing the Kroxylicious Operator
+
+[role="_abstract"]
+This procedure shows how to install the Kroxylicious Operator in your Kubernetes cluster.
+
+.Prerequisites
+
+* You need an account with permission to create and manage `CustomResourceDefinition` and RBAC (`ClusterRole`) resources.
+* You have downloaded one of {OperatorAssetZipFileName} or {OperatorAssetTgzFileName} and extracted the contents into the current directory.
+
+.Procedure
+
+. Edit the Kroxylicious installation files to use the namespace the operator is going to be installed into.
++
+For example, in this procedure the operator is installed into the namespace `my-kroxylicious-operator-namespace`.
++
+include::../../snippets/snip-operator-namespace-sed.adoc[]
```

Comment:
out of curiosity why is this pulled out to a separate snippet file?

Reply:
No idea. @k-wall ?